lib/pages/settings/view/settings_page.dart

```diff
@@ -141,6 +157,16 @@ class _SettingsPageState extends State<SettingsPage> {
                                         color: Theme.of(context).disabledColor,
                                       )
                                   : Theme.of(context).textTheme.bodyText1),
+                        ),
+                        const Divider(),
+                        Text(S.of(context).labelVersion,
+                            textAlign: TextAlign.center,
+                            style: Theme.of(context).textTheme.bodyText1),
+                        Text(_packageInfo.version,
+                            textAlign: TextAlign.center,
+                            style: Theme.of(context).textTheme.bodyText1),
```

Comment:
I'd make it one line. If you disagree, feel free to bring it up for discussion in the group chat and we'll see what the majority thinks.

Reply:
Why did you mark as resolved? 🤔